sonic_platform_base/sfp_base.py

```diff
@@ -304,3 +304,47 @@ def set_power_override(self, power_override, power_set):
             False if not
         """
         raise NotImplementedError
+
+    def read_eeprom_devid(self, port_num, devid, offset, num_bytes):
```

Comment:
There should be no need for port_num or devid here. If the platform uses sysfs/i2c to communicate with the EEPROM, the vendor will derive those based on which transceiver this function is called on and can in turn call the functions in sfputilbase.py.

Reply:
fixed